common/management/commands/check_db_connection.py

```diff
@@ -7,31 +7,50 @@
 
 
 class Command(BaseCommand):
-    help = 'This command checks availability of configured database.'
+    help = "This command checks availability of configured database."
 
     requires_system_checks = False
 
     def add_arguments(self, parser):
-        parser.add_argument('--count', type=int, default=5, help='Number of connections attempts before failing')
-        parser.add_argument('--sleep', type=int, default=1, help='Seconds to wait after every failed attempt')
+        parser.add_argument(
+            "--count",
+            type=int,
+            default=5,
+            help="Number of connections attempts before failing",
+        )
+        parser.add_argument(
+            "--sleep",
+            type=int,
+            default=1,
+            help="Seconds to wait after every failed attempt",
+        )
 
     def handle(self, *args, **options):
-        if 0 > options['count']:
-            raise CommandError(u'No negative count allowed. '
-                               u'It''s not I''m picky, but I simply don''t now, how to handle it.')
-        if 0 > options['sleep']:
-            raise CommandError(u'No negative sleep allowed. Bear with me, but I forgot my time machine.')
+        if 0 > options["count"]:
+            raise CommandError(
+                "No negative count allowed. "
+                "It"
+                "s not I"
+                "m picky, but I simply don"
+                "t now, how to handle it."
```

Comment:
```suggestion
                "No negative count allowed. It's not I'm picky, "
                "but I simply don't now, how to handle it."
```

Not only it's almost unreadable, disappearing single quotes make error message ugly.

Reply:
Good spotting! Technically, there were no apostrophes in the first place before but just a bunch of concatenated strings (and black was right to fix it this way). o_O

Reply:
I guess you're right. Did older python version maybe treat double-single-quotes as "escaped"? Or did nobody ever noticed, error message is wrong?

Whatever, my inner Monk is happy, this is fixed - and it's not 5 silly lines. 😆